charts/operator/crds/monitoring.googleapis.com_clusterrules.yaml

```diff
@@ -124,6 +124,44 @@ spec:
             type: object
           status:
             description: Most recently observed status of the resource.
+            properties:
```

Comment:
What caused this piece to be generated? I don't see a matching change.

Reply:
This is the addition of the `MonitoringCondition` field.

Reply:
`MonitoringCondition` seems to have been added in a prior release. So it's strange that this is showing up now.

Reply:
The definition is copied into each place it's used. This is the first time it's used in `ClusterRules`!